### ISSUE TITLE
error: Failed to find platform for ARM64 as Linux Userspace

### ISSUE THREAD

**Issue by nderjung** [unikraft/unikraft#352]
### Describe the bug

Attempting to compile [helloworld](https://github.com/unikraft/app-helloworld) for ARM64 as Linux Userspace application results in invalid platform-architecture combination.

### Steps to reproduce

 * `CONFIG_ARCH_ARM_64=y`
 * `CONFIG_PLAT_LINUXU=y`
 * `CONFIG_LIBPOSIX_LIBDL=y`
 * `CONFIG_LIBPOSIX_PROCESS=y`
 * `CONFIG_LIBPOSIX_USER=y`
 * `CONFIG_LIBPOSIX_SYSINFO=y`
 * `CONFIG_LIBVFSCORE=y`
 * `CONFIG_LIBUKSWRAND=y`
 * `CONFIG_LIBUKSP=y`
 * `CONFIG_LIBUKBLKDEV=y`

### Which architectures were you using or does this bug affect?

arm64

### Which platforms were you using or does this bug affect?

linuxu

### Relevant log output

```shell
/tmp/build/80754af9/built/apps/helloworld/build/Makefile:662: You did not choose any target platform.
/tmp/build/80754af9/built/apps/helloworld/build/Makefile:663: *** Please choose at least one target platform in the configuration!.  Stop.
make: *** [Makefile:1015: sub-make] Error 2
make: Leaving directory '/tmp/build/80754af9/built/unikraft'
make: Entering directory '/tmp/build/80754af9/built/unikraft'
/tmp/build/80754af9/built/apps/helloworld/build/Makefile:662: You did not choose any target platform.
/tmp/build/80754af9/built/apps/helloworld/build/Makefile:663: *** Please choose at least one target platform in the configuration!.  Stop.
make: *** [Makefile:1015: sub-make] Error 2
make: Leaving directory '/tmp/build/80754af9/built/unikraft'
make: Entering directory '/tmp/build/80754af9/built/unikraft'
/tmp/build/80754af9/built/apps/helloworld/build/Makefile:662: You did not choose any target platform.
/tmp/build/80754af9/built/apps/helloworld/build/Makefile:663: *** Please choose at least one target platform in the configuration!.  Stop.
make: *** [Makefile:1015: sub-make] Error 2
make: Leaving directory '/tmp/build/80754af9/built/unikraft'
```

**Comment by skuenzer** [comment]
Right, the `linuxu` platform does not yet support Arm64 but it should be a minor thing to get it working.

**Comment by JinmoSung** [comment]
> Right, the `linuxu` platform does not yet support Arm64 but it should be a minor thing to get it working.

Is there any plan when linuxu platform will support Arm64?

**Comment by razvand** [comment]
@JinmoSung , yes, we plan to add support for that in the next week or so. @maniatro111 is working on it.

**Comment by flpostolache** [comment]
@nderjung Can this issue be closed now? I've added support for linuxu-arm64 in this [pr](https://github.com/unikraft/unikraft/pull/418). The problem is now that `vfscore` uses `ukshched` which hides the option to compile for `linux` on `arm64`.

**Comment by nderjung** [comment]
Fixed in #418
